### PR TITLE
Add Python script for updating field dimensions across the project

### PR DESCRIPTION
**Pull request by shueja**
Mostly unused. A near-future PR will use the values from autogenerated source wherever the current field dimensions are needed.

### CODE REVIEW

File: src/components/field/svg/fields/FieldDimensions.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Auto-generated by update_field_dimensions.py
+export const FIELD_LENGTH = 16.5811;
```
Comment (spacey-sooty) [inline src/components/field/svg/fields/FieldDimensions.tsx:2]: These should be used for the field keep in constraint

File: update_field_dimensions.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+class FieldDimensions {{
+  static final double FIELD_LENGTH = {length};
+  static final double FIELD_WIDTH = {width};
+}}""",
+    ),
+    # Python ChoreoLib
+    Location(
+        relative_path=Path("choreolib/py/choreo/util/field_dimensions.py"),
+        template=lambda length, width: f"""# Auto-generated by update_field_dimensions.py
+FIELD_LENGTH = 16.5811
```
Comment (spacey-sooty) [inline update_field_dimensions.py:46]:
```suggestion
FIELD_LENGTH = {length}
```

File: update_field_dimensions.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+FIELD_WIDTH = 8.19912""",
```
Comment (spacey-sooty) [inline update_field_dimensions.py:47]:
```suggestion
FIELD_WIDTH = {length}""",
```

**Review by spacey-sooty — CHANGES_REQUESTED** [review]

File: src/components/field/svg/fields/FieldDimensions.tsx (hunk 1 above)
Comment (shueja) [reply src/components/field/svg/fields/FieldDimensions.tsx:2]: The keep-in constraint will derive its values from this file. That's part of the near-future PR mentioned above.

**Review by spacey-sooty — APPROVED** [review]

**Pull request merged**